### PR TITLE
Added script for getting management certificates in subscription

### PR DESCRIPTION
**Pull request by HarshitArora8**

### CODE REVIEW

File: Scripts/RemediationScripts/Get-ManagementCertificates.ps1 [hunk 1]
```diff
@@ -0,0 +1,352 @@
+<###
+# Overview:
+    This script is used to get Management Certificates in a Subscription.
+
+# Prerequisites:
+    1. Co-Admin or higher privileges in a Subscription.
```
Comment (TarunKrShukla) [inline Scripts/RemediationScripts/Get-ManagementCertificates.ps1:6]: Just call out Co-Administrator role is required.

File: Scripts/RemediationScripts/Get-ManagementCertificates.ps1 [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+
+    # Using Get-AzAccessToken to get access token for fetching management certificates.
+    $ClassicAccessToken= (Get-AzAccessToken -ResourceUrl $ResourceAppIdURI).Token
+    
+    Write-Host $([Constants]::DoubleDashLine)
+    Write-Host "[Step 2 of 2] Fetching  management certificates in Subscription: [$($SubscriptionId)]"
+    Write-Host $([Constants]::SingleDashLine)
+
+    if($null -ne $ClassicAccessToken)
+    {
```
Comment (TarunKrShukla) [inline Scripts/RemediationScripts/Get-ManagementCertificates.ps1:162]: Just log message there was issue while fetching mgt cert in this case as we can't proceed w/o access token.

File: Scripts/RemediationScripts/Get-ManagementCertificates.ps1 [hunk 3]
```diff
@@ -0,0 +1,352 @@
...
+        $header = "Bearer " + $ClassicAccessToken
+        $headers = @{"Authorization"=$header;"Content-Type"="application/json"; "x-ms-version" ="2013-08-01"}
+
+        $uri = [string]::Format("{0}/{1}/certificates",$ResourceAppIdURI,$SubscriptionId)
+        $mgmtCertsResponse = Invoke-WebRequest -Method GET -Uri $uri -Headers $headers -UseBasicParsing
+        if($mgmtCertsResponse.StatusCode -ge 200 -and $mgmtCertsResponse.StatusCode -le 399)
```
Comment (TarunKrShukla) [inline Scripts/RemediationScripts/Get-ManagementCertificates.ps1:168]: Same if response was not successful just log message that error occurred while fetching mgt certs.

File: Scripts/RemediationScripts/Get-ManagementCertificates.ps1 [hunk 4]
```diff
@@ -0,0 +1,352 @@
...
+        {
+            if($null -ne $mgmtCertsResponse.Content)
+            {
+                [xml] $mgmtCerts = $mgmtCertsResponse.Content;
+                if($null -ne $mgmtCerts -and [Helpers]::CheckMember($mgmtCerts, "SubscriptionCertificates.SubscriptionCertificate"))
+                {
+                    # Includes all the management certificates from the subscription
+                    $ManagementCertificates = [ManagementCertificate]::ListManagementCertificates($mgmtCerts.SubscriptionCertificates.SubscriptionCertificate)
+                
+                    $context = Get-AzContext
```
Comment (TarunKrShukla) [inline Scripts/RemediationScripts/Get-ManagementCertificates.ps1:178]: Why context check is needed here

File: Scripts/RemediationScripts/Get-ManagementCertificates.ps1 (hunk 1 above)
Comment (HarshitArora8) [reply Scripts/RemediationScripts/Get-ManagementCertificates.ps1]: Added.

File: Scripts/RemediationScripts/Get-ManagementCertificates.ps1 (hunk 2 above)
Comment (HarshitArora8) [reply Scripts/RemediationScripts/Get-ManagementCertificates.ps1:162]: Added.

File: Scripts/RemediationScripts/Get-ManagementCertificates.ps1 (hunk 3 above)
Comment (HarshitArora8) [reply Scripts/RemediationScripts/Get-ManagementCertificates.ps1:168]: Added.

File: Scripts/RemediationScripts/Get-ManagementCertificates.ps1 (hunk 4 above)
Comment (HarshitArora8) [reply Scripts/RemediationScripts/Get-ManagementCertificates.ps1]: Removed AzContext check as it is already done above.

**Review by TarunKrShukla — APPROVED** [review]
Looks good

**Pull request merged**